web/src/game/game-details.html

```diff
@@ -79,9 +82,10 @@
   </ghvz-super-form>
 
   <div hidden$="[[!game]]">
-    <div id="number">ID: <b>[[game.id]]</b></div>
-    <div id="name">Name: <b>[[game.name]]</b></div>
-    <div id="stunTimer">Stun Timer: <b>[[game.stunTimer]]</b></div>
+    <!-- Would it break anything to give the two elements diff names?-->
```

Comment:
Ignore that comment... at some point the game ID and name had the same id, I was wondering if I should change that, but accidentally wrote name instead of id. Anyway, I guess someone else changed it - all good now!